qemu/tests/cfg/win_virtio_single_driver.cfg

```diff
@@ -0,0 +1,50 @@
+-win_virtio_single_driver:
+    type = win_virtio_single_driver
+    only Windows
+    driver_repo_path = C:\Windows\System32\DriverStore\FileRepository
+    tmp_folder = C:\tmp
+    driver_uninstall_command = 'cmd /c C:\dpinst.exe /u %s\%s\%s.inf /D /SA /C /F /S'
+    driver_install_command = '%s:\install_driver.bat %s\%s'
+    variants:
+        -with_netkvm:
+            driver_name = netkvm
+            nics += " nic2"
+            nic_model_nic2 = virtio
+            nic_model_nic1 = rtl8139
+        -with_viorng:
+            driver_name = viorng
+            no_virtio_rng:
+                virtio_rngs += " rng0"
+                backend_rng0 = rng-random
+                backend_type = passthrough
+                filename_passthrough = /dev/random
+        -with_block:
+            driver_name = viostor
+            drive_format_image1 = ide
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = virtio
+        -with_vioscsi:
+            driver_name = vioscsi
+            cd_format_fixed =ide
+            drive_format_image1 = ide
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = scsi-hd
+        -with_vioserial:
+            driver_name = vioser
+            virtio_ports = "vs"
+            virtio_port_type = serialport
+        -with_balloon:
+            driver_name = balloon
+            balloon = balloon0
+            balloon_dev_devid = balloon0
+            balloon_dev_add_bus = yes
+    variants:
+        -reinstall:
+            downgrade = False
+        -downgrade:
+            fix_str = driver_install_cmd_
+            downgrade = True
```

Comment:
Hi @vipmike007 

I think there should be the cfg for driver_install, you can add internal cfg to cover it.
If there is no cfg for the driver_install, the case would fail.

Thanks.